modules/distr-tracing-tempo-configuring-tempooperator-metrics-and-alerts.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// * distr-tracing-tempo-configuring.adoc
+
+:_content-type: PROCEDURE
+[id="configuring-tempooperator-metrics-and-alerts_{context}"]
+= Configuring {TempoOperator} metrics and alerts
+
+When installing the {TempoOperator} by using the OpenShift web console, you can select the *Enable Operator recommended cluster monitoring on this Namespace* checkbox, which enables creating metrics and alerts of the {TempoOperator}.
```

Comment:
```suggestion
When installing the {TempoOperator} by using the {product-title} web console, you can select the *Enable Operator recommended cluster monitoring on this Namespace* checkbox, which enables creating metrics and alerts of the {TempoOperator}.
```
Related: https://github.com/openshift/openshift-docs/pull/62410#discussion_r1318989879

Reply:
Thank you. Fixed